MastodonSDK/Sources/MastodonLocalization/Resources/en.lproj/Localizable.strings

```diff
@@ -154,7 +154,7 @@ Your profile looks like this to them.";
 "Common.Controls.Timeline.Timestamp.Now" = "Now";
 "Scene.AccountList.AddAccount" = "Add Account";
 "Scene.AccountList.DismissAccountSwitcher" = "Dismiss Account Switcher";
-"Scene.AccountList.TabBarHint" = "Current selected profile: %@. Double tap then hold to show account switcher";
+"Scene.AccountList.SwitchAccounts" = "Switch Accounts";
```

Comment:
That will not take effect. Suggest rollback changes.

```suggestion
"Scene.AccountList.TabBarHint" = "Current selected profile: %@. Double tap then hold to show account switcher";
```

Reply:
Done!